cmd/juju/application/bundle.go

```diff
@@ -183,6 +184,9 @@ type bundleHandler struct {
 	// data is the original bundle data that we want to deploy.
 	data *charm.BundleData
 
+	// The bundle URL from the charmstore.
```

Comment:
// bundleURL is the URL of the bundle when deploying a bundle from the char store, nil otherwise.
Or similar?

Reply:
done